crates/sui-indexer-alt/src/db/mod.rs

```diff
@@ -30,13 +34,13 @@ pub struct DbConfig {
     database_url: Url,
 
     /// Number of connections to keep in the pool.
-    #[arg(long, default_value_t = 100)]
+    #[arg(long, default_value = stringify!(DEFAULT_POOL_SIZE))]
```

Comment:
```suggestion
    #[arg(long, default_value_t = DEFAULT_POOL_SIZE)]
```

Reply:
thanks for the review @bmwill, on the other thread, Ashok and I were wondering if there is a good way to define the default value twice here although they share const like DEFAULT_POOL_SIZE
> main thing I'd like to see is whether we can avoid duplicating the config defaults (cc @bmwill in case you have any tricks here...)

any wisdom?

Reply:
Unfortunately no, we'll have to have it here as well as in the Default impl. But we could always add a test to ensure they stay in sync